src/components/MeasurementOption/MeasurementOption.js

```diff
@@ -2,14 +2,23 @@ import React from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { withTranslation } from 'react-i18next';
-
-import MeasurementsDropdown from 'components/MeasurementsDropdown';
+import i18next from 'i18next';
 
 import selectors from 'selectors';
 
 import './MeasurementOption.scss';
 
-class MeasurementOption extends React.PureComponent {
+class MeasurementOption extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { currScaleFrom: props.scale[0][0],
+      currUnitFrom: props.scale[0][1],
+      currScaleTo: props.scale[1][0],
+      currUnitTo: props.scale[1][1],
+      currPrecision: props.precision,
+      isEditing: false };
+  }
+
   static propTypes = {
```

Comment:
Could we move propTypes to the top of the class just like other components?

Reply:
Resolved in 084caf3. Thanks!